docs/workflows/common-workflow-steps/build-and-test/add-badge-app-icon.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Add a Badge to Your App Icon
+metaTitle: Add a Badge to Your App Icon
+metaDescription: Learn how to add a badge to your app icon with Appcircle's badge component.
```

Comment:
`metaTitle` and `metaDescription` are not working for SEO. Use the `description` prop instead

```suggestion
description: Learn how to add a badge to your app icon with Appcircle's badge component.
```